heartbeat/Jenkinsfile.yml

```diff
@@ -35,8 +35,16 @@ stages:
             branches: true     ## for all the branches
             tags: true         ## for all the tags
         stage: extended
-    build:
-        mage: "mage build test"
+    unitTest:
+        mage: "mage build unitTest"
+        stage: mandatory
+    goIntegTest:
+        mage: "mage goIntegTest"
+        withModule: true
+        stage: mandatory
+    pythonIntegTest:
+        mage: "mage pythonIntegTest"
+        withModule: true
```

Comment:
Same here, not sure if we can use `withModule`. Heartbeat doesn't have modules, it has monitors, but they have a different directory layout.